content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -566,60 +566,142 @@ it.skip('returns "fizz" when number is multiple of 3', () => {
 
 <Alert type="success">
 
-<Icon name="check-circle" color="green"></Icon> **Best Practice:** Clean up
-state **before** tests run.
+<Icon name="check-circle" color="green"></Icon> **Best Practice:** Tests should
+always be able to be run independently from one another **and still pass**.
```

Comment:
I think this captures the idempotency talked about in https://github.com/cypress-io/cypress/pull/24316#discussion_r1002060664